client/camera/CameraVideo.js

```diff
@@ -122,6 +122,7 @@ export default class CameraVideo extends React.Component {
             return (
               <Knob
                 key={position}
+                label={["BR", "BL", "TL", "TR"][position]}
```

Comment:
This seems the wrong way around — maybe move [`cornerNames`](https://github.com/janpaul123/paperprograms/blob/19204bcc66c3ba3b19efd1d73b038cb1087b5ed7/client/camera/detectPrograms.js#L22) into `constants.js` and use that here?